src/Stegside/AvtaleContext.tsx

```diff
@@ -276,7 +282,9 @@ export class TempAvtaleProvider extends React.Component<any, State> {
         return (
             <AvtaleContext.Provider value={context}>
                 {this.state.feilmelding && (
```

Comment:
Feilmelding er ikke optional lengre så trenger vel ikke sjekke om den eksisterer.

Reply:
Den er enten tom string (falsy) eller har en feilmelding, som gjør at den rendrer Varsel-komponent. Forslag til andre måter å toggle Varsel-komponenten?

Reply:
Jeg synes dette er en OK å toggle på, det blir mer kompleksistet hvis man skal slenge på et ekstra flagg for visning.

Reply:
Men kanskje det er mer beskrivende med "fjernFeilmelding" i stedet for "skjulFeilmelding"?